test/vanilla/legacy/Expected/AcceptanceTests/PackageModeDataPlane/packagemode/_vendor.py

```diff
@@ -0,0 +1,17 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
```

Comment:
nit: paging is not a vanilla test, it's an azure test. it'd be better to use a swagger that is a vanilla swagger for packagemodedataplane, since we don't test with azure deps in vanilla

Reply:
I change to use `any-type.json` which is vanilla swagger